emba.sh

```diff
@@ -63,15 +65,16 @@ main()
   export SHORT_PATH=0           # short paths in cli output
   export ONLY_DEP=0             # test only dependency
   export FORCE=0
+  export LOG_GREP=0
   export QEMULATION=0
   export PRE_CHECK=0            # test and extract binary files with binwalk
                                 # afterwards do a default emba scan
 
-  export LOG_DIR="./logs"
-  export CONFIG_DIR="./config"
-  export EXT_DIR="./external"
-  export HELP_DIR="./helpers"
-  export MOD_DIR="./modules"
+  export LOG_DIR="$INVOCATION_PATH""/logs"
+  export CONFIG_DIR="$INVOCATION_PATH""/config"
+  export EXT_DIR="$INVOCATION_PATH""/external"
+  export HELP_DIR="$INVOCATION_PATH""/helpers"
+  export MOD_DIR="$INVOCATION_PATH""/modules"
```

Comment:
1st - cosmetic updates like this (we can accept this)

Reply:
This is not only cosmetic: I needed that for this feature.